DESCRIPTION

```diff
@@ -55,7 +55,6 @@ Suggests:
     knitr,
     lubridate,
     readxl,
```

Comment:
Can we remove readxl now that the NBER file being fetched is a CSV?

Reply:
yes, sorry, I missed that one.

Reply:
good catch